src/main/scala/org/broadinstitute/dsde/workbench/sam/db/tables/UserAttributesTable.scala

```diff
@@ -4,13 +4,25 @@ import org.broadinstitute.dsde.workbench.model._
 import org.broadinstitute.dsde.workbench.sam.db.SamTypeBinders
 import org.broadinstitute.dsde.workbench.sam.model.api.SamUserAttributes
 import scalikejdbc._
-
 import java.time.Instant
 
 final case class UserAttributesRecord(
     samUserId: WorkbenchUserId,
     marketingConsent: Boolean,
-    updatedAt: Instant
+    firstName: Option[String],
+    lastName: Option[String],
+    organization: Option[String],
+    contactEmail: Option[String],
+    title: Option[String],
+    department: Option[String],
+    interestInTerra: Option[List[String]],
+    programLocationCity: Option[String],
+    programLocationState: Option[String],
+    programLocationCountry: Option[String],
+    researchArea: Option[List[String]],
+    additionalAttributes: Option[String],
+    createdAt: Option[Instant],
```

Comment:
I did make both of these timestamp fields optional to account for users who already have a record in the DB without a `createdAt` date, and to allow for (slightly) easier testing